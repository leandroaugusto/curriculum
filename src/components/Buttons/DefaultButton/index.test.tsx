import { render, screen } from "@testing-library/react";
import Button from ".";

const mockOnClick = jest.fn();

describe("Button", () => {
  it("renders the button component", () => {
    const { container } = render(
      <Button handleClick={mockOnClick} id="button-id" label="Button" />
    );

    const button = screen.getByTestId("button-id-button");

    expect(button).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
