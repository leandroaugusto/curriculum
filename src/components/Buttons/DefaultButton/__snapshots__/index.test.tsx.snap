// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders the button component 1`] = `
<div>
  <button
    class="btn undefined"
    data-testid="button-id-button"
    id="button-id"
  >
    Button
  </button>
</div>
`;
